### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][JAWS/NVDA][MAS4.2.1][IntelliJ][Wipro]: NVDA and Jaws are  not announcing the content present in  Create Authentication files Window sequentially . 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1510]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Open your project with IntelliJ IDEA
3. Tab to Tools menu, point to "Azure", and then select "Azure Sign In".
4.  In the Azure Sign In window, select "Automated" radio button, and navigate to "New" button and select it.
5. In the Azure Login Dialog window, enter your "Azure credentials", and then select "Sign in" button.
6. After sign In Create Authentication Files window will opened .
7. In "Create Authentication files window" Press "Insert+B" to read  the entire window then observe the NVDA and Jaws Behavior

**Expected Result:**

NVDA and Jaws Should announce in the following  manner

- Create Authentication Files  dialog  
- A new Active Directory service principal representing this IDE will be created as needed and as allowed by your access permissions.
 The service principal will be granted Contributor-level access to each selected 
  Select the subscription(s) to create credentials for:
- table
  Subscription name header
  Subscription id header
  check box  checked
  DevDiv Key Vault
  bd62906c-0a81-43c3-a2f8-126e4cf66ada
  check box  checked
  HDInsight VS Tool Team
  7f31cba8-b597-4129-b158-8f21a7395bd0
- scroll bar  
- Destination folder:
- edit
  C:\Users\v-swbi
- Browse  button  Shift+Enter
- Start  button
- Cancel  button

**Actual Result:**
**NVDA Behavior:**
NVDA is announicng as
Create Authentication Files  dialog  Destination folder:
Select the subscription(s) to create credentials for:
table
subscription.
edit  text/plain
subscription.
NVDA+b
Create Authentication Files  dialog  Destination folder:
Select the subscription(s) to create credentials for:
table
check box  checked
DevDiv Key Vault
bd62906c-0a81-43c3-a2f8-126e4cf66ada
check box  checked
HDInsight VS Tool Team
7f31cba8-b597-4129-b158-8f21a7395bd0
check box  checked
Java Dev Experience Tests with TTL = 7 Days
ffa52f27-be12-4cad-b1ea-c2c241b6cceb
scroll bar  0
Subscription name
Subscription id
edit
C:\Users\v-swbi
Browse  button  Shift+Enter
Destination folder:
edit  text/plain
subscription.
Select the subscription(s) to create credentials for:
filler
Start  button
Cancel  button
**Jaws Behavior:**
Create Authentication Files dialog
Subscription name Subscription id C:\Users\abby Destination folder: A new Active Directory service principal representing this IDE will be created as needed and as allowed by your access permissions.
The service principal will be granted Contributor-level access to each selected subscription. Select the subscription(s) to create credentials for: 
table
Not Selected CIS_BVT_Helsinki

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)

![hierarchy is incorrect for create authentication files dialog](https://user-images.githubusercontent.com/22956130/36149477-d6b2c0da-10e5-11e8-9b3a-e415485c051c.jpg)
![create authentication files-table](https://user-images.githubusercontent.com/22956130/36149485-e017a924-10e5-11e8-83e6-73561e2d3f38.jpg)

**Comment by Eskibear** [comment]
under further investigation

**Comment by Eskibear** [comment]
Cannot reproduce it.

According to the practice, NVDA cannot read content of any dialog in IntelliJ IDEA 2017.3.4 Community.
A ticket is created to track the issue.

Ref:
https://youtrack.jetbrains.com/issue/IDEA-191325

**Comment by sallyur** [comment]
As mentioned above, in our practice we cannot reproduce the issue, as NVDA cannot read content of IntelliJ IDEA at all.

- Importance of experience: it depends, varies from somewhat important to important.
- Quality of experience: not impacted. Since the Narrator is not experiencing this kind of issues, users can use the Narrator to as an alternative workaround.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749985
